sloglint.go

```diff
@@ -103,11 +112,13 @@ func flags(opts *Options) flag.FlagSet {
 	boolVar(&opts.NoRawKeys, "no-raw-keys", "enforce using constants instead of raw keys")
 	strVar(&opts.KeyNamingCase, "key-naming-case", "enforce a single key naming convention (snake|kebab|camel|pascal)")
 	boolVar(&opts.ArgsOnSepLines, "args-on-sep-lines", "enforce putting arguments on separate lines")
+	repeatedStrVar(&opts.ForbiddenKeys, "forbidden-key", "enforce not using specific keys. Can be repeated or comma separated")
 
 	return *fset
 }
 
 var slogFuncs = map[string]int{ // funcName:argsPos
+	"log/slog.With":                   0,
```

Comment:
Is this change related to the new option?

Reply:
Yes. You can append attributes with the `With` function.

Example: https://go.dev/play/p/PEBgTYRVy2M

```go
package main

import "log/slog"

func main() {
	log := slog.Default()
	log.With("key", "value").Info("test_str", "other_key", "other_value")
}

```

```text
2009/11/10 23:00:00 INFO test_str key=value other_key=other_value
```

Loook that there is `key=value` and `other_key=other_value`

Reply:
Looks like another escape was recently identified in https://github.com/go-simpler/sloglint/issues/38. Perhaps we can tack the missing `With` onto that and get all the options updated in one pass?

Reply:
I split it into https://github.com/go-simpler/sloglint/pull/41